lib/Quantization/Quantization.cpp

```diff
@@ -388,6 +512,33 @@ class FunctionQuantizer : public FunctionConverter {
       dequantize->setNthInput(DequantizeNode::InputIdx, rescale);
       break;
     }
+
+    casesForIntLookupTableReplacement : {
```

Comment:
I was matching the previous style here -- we have another one of these in this file, called `casesForSingleMatchingInOutType`. I could change `casesForIntLookupTableReplacement` as suggested, but I'm not sure I can get the other one to the same format, since the other one includes static asserts to verify the cases all have matching formats. And I think I'd prefer them to have matching style. LMK what you think.